src/components/LegoGallery/index.tsx

```diff
@@ -0,0 +1,157 @@
+import ThemedImage from '@theme/ThemedImage';
+import clsx from 'clsx';
+
+const STANDARDS = [
+  {
+    title: 'LSP0 - ERC725 Account',
+    link: './universal-profile/lsp0-erc725account',
+    lightImage: '/img/standards/legos/LSP0-lego.png',
+    darkImage: '/img/standards/legos/LSP0-lego.png',
+    text: 'A blockchain based account that can own assets and be used by people, organizations, etc...',
+  },
+  {
+    title: 'LSP1 - Universal Receiver',
+    link: './generic-standards/lsp1-universal-receiver',
+    lightImage: '/img/standards/legos/LSP1-lego.png',
+    darkImage: '/img/standards/legos/LSP1-lego.png',
+    text: 'Allows to receive notifications about incoming and outgoing transactions or informations.',
+  },
+  {
+    title: 'LSP2 - ERC725Y JSON Schema',
+    link: './generic-standards/lsp2-json-schema',
+    lightImage: '/img/standards/legos/LSP2-lego.png',
+    darkImage: '/img/standards/legos/LSP2-lego.png',
+    text: 'A schema to describe how ERC725Y data key-value pairs can be interpreted and encoded/decoded.',
+  },
+  {
+    title: 'LSP3 - Profile Metadata',
+    link: './universal-profile/lsp3-profile-metadata',
+    lightImage: '/img/standards/legos/LSP3-lego.png',
+    darkImage: '/img/standards/legos/LSP3-lego.png',
+    text: 'A set of ERC725Y data keys useful to describe a smart contract based profile.',
+  },
+  {
+    title: 'LSP4 - Digital Asset Metadata',
+    link: './tokens/LSP4-Digital-Asset-Metadata',
+    lightImage: '/img/standards/legos/LSP4-lego.png',
+    darkImage: '/img/standards/legos/LSP4-lego.png',
+    text: 'A set of ERC725Y data keys useful to describe the metadata of a digital asset.',
+  },
+  {
+    title: 'LSP5 - Received Assets',
+    link: './universal-profile/lsp5-received-assets',
+    lightImage: '/img/standards/legos/LSP5-lego.png',
+    darkImage: '/img/standards/legos/LSP5-lego.png',
+    text: 'Defines a set of ERC725Y data keys to store the addresses of assets received.',
+  },
+  {
+    title: 'LSP6 - Key Manager',
+    link: './universal-profile/lsp6-key-manager',
+    lightImage: '/img/standards/legos/LSP6-lego.png',
+    darkImage: '/img/standards/legos/LSP6-lego.png',
+    text: 'A smart contract and a set of predefined permissions to control an account or any smart contract through multiple addresses.',
+  },
+  {
+    title: 'LSP7 - Digital Asset',
+    link: './tokens/LSP7-Digital-Asset',
+    lightImage: '/img/standards/legos/LSP7-lego.png',
+    darkImage: '/img/standards/legos/LSP7-lego.png',
+    text: 'A standard interface for digital assets that can be either fungible or non-fungible tokens.',
+  },
+  {
+    title: 'LSP8 - Identifiable Digital Asset',
+    link: './tokens/LSP8-Identifiable-Digital-Asset',
+    lightImage: '/img/standards/legos/LSP8-lego.png',
+    darkImage: '/img/standards/legos/LSP8-lego.png',
+    text: 'A standard interface to represent identifiable digital assets, allowing tokens and NFTs to be uniquely traded and given specific metadata.',
```

Comment:
I would suggest capping all description texts to `75` or `80` characters. This way, they all fit within `3` lines of a 1/4 card. I think it is unnecessary to describe them perfectly here, as users will get the full description after clicking right on it. However, it will remove lots of white space from the page.